sphinx_gallery/gen_rst.py

```diff
@@ -495,14 +495,11 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf):
 
     time_elapsed = 0
 
-    ref_fname = example_file.replace(os.path.sep, '_')
+    ref_fname = example_file.replace(op.sep, '_')
     example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
 
-    if not fname.startswith('plot'):
-        convert_func = dict(code=codestr2rst, text=text2string)
-        for blabel, bcontent in script_blocks:
-            example_rst += convert_func[blabel](bcontent) + '\n'
-    else:
+    filename_pattern = gallery_conf.get('filename_pattern', op.sep + 'plot')
+    if re.search(filename_pattern, op.realpath(op.join(src_dir, fname))):
```

Comment:
On Fri, Jan 8, 2016 at 8:52 PM, Loïc Estève <notifications@github.com>
wrote:

> In sphinx_gallery/gen_rst.py
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/85#discussion_r49198242>
> :
>
> >      example_rst = """\n\n.. _sphx_glr_{0}:\n\n""".format(ref_fname)
> >
> > -    if not fname.startswith('plot'):
> > -        convert_func = dict(code=codestr2rst, text=text2string)
> > -        for blabel, bcontent in script_blocks:
> > -            example_rst += convert_func[blabel](bcontent) + '\n'
> > -    else:
> > +    filename_pattern = gallery_conf.get('filename_pattern', op.sep + 'plot')
> > +    if re.search(filename_pattern, op.realpath(op.join(src_dir, fname))):
>
> Maybe abspath rather than realpath? The differences seems to be how it
> treats symbolic links. It feels abspath is more what we want here.
>
fixed.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sphinx-gallery/sphinx-gallery/pull/85/files#r49198242>
> .
>